### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by hjae78** [GregTechCE/GregTech#1660]
**Describe the bug**
AdvancedTextWidget do not center their texts.

**Setup**
Dev

**Steps To Reproduce**
1) Look at Fluid Regulators GUI
2) Observe that the text isn't centered.

**Expected behavior**
That the text should be centered.

**Screenshots**

![image](https://user-images.githubusercontent.com/54247183/120917788-6fac3580-c6b1-11eb-9ebf-471f9a731f8a.png)

![image](https://user-images.githubusercontent.com/54247183/120917790-72a72600-c6b1-11eb-961d-1c914d2d6f81.png)

**Comment by Archengius** [comment]
And it's intended. In multiblock screens for example, text is not supposed to be centered

**Comment by hjae78** [comment]
> And it's intended. In multiblock screens for example, text is not supposed to be centered

Good point, I was thinking of this too. Maybe a boolean to control it when desired?
